src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
```

Comment:
IMO text should not be a property, not sure if this was discussed. Ideally I would say this Text component act nearly like a `<p>` just with explicitly enumerated options for styling.

Ex.

```js
<Text bold italic underline>
   paragraph content goes here
</Text>
```